File: Comparativa Python/Ejercicio4-Arrays.py
"""4. Escribe un programa que defina un vector de números y muestre en la
salida estándar el vector en orden inverso - del último al primer elemento."""

miLista = [1, 2, 3, 4, 5]

for i in range(len(miLista)):
    print(miLista[-i-1], ' ', end="")

File: Arreglos o Vectores/Ejercicio4-Arrays.cpp
/*4. Escribe un programa que defina un vector de números y muestre en la salida estándar el vector en orden inverso - del último al primer elemento.*/

#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    int numeros[] = {1,2,3,4,5};

    for (int i=4; i>=0; i--){
        cout<<"["<<i<<"] -> "<<numeros[i]<<endl;
    }
    system("pause");
    return 0;
}